### PR TITLE
Support ime's chained to modifiers

### PR DESCRIPTION
**Pull request by NduatiK**
Support `Color(...).opacity(0.25)` in

```swift
foregroundStyle(Color(.displayP3, red: 0.4627, green: 0.8392, blue: 1.0).opacity(0.25))
```

Resolves https://github.com/liveview-native/live_view_native_stylesheet/issues/41

### CODE REVIEW

**Comment by NduatiK** [comment]
@carson-katri, please confirm the output AST in the test is correct

File: lib/live_view_native/swiftui/rules_parser/post_processors.ex [hunk 1]
```diff
@@ -82,6 +64,7 @@ defmodule LiveViewNative.SwiftUI.RulesParser.PostProcessors do
 
   def to_scoped_ime_ast(rest, [[nil, variable], scope], context, {_line, _}, _byte_offset) do
     {rest, [variable, String.to_atom(scope)], context}
+    # |> IO.inspect()
```
Comment (carson-katri) [inline lib/live_view_native/swiftui/rules_parser/post_processors.ex:67]: Can this be removed?
```suggestion
```

**Review by carson-katri — CHANGES_REQUESTED** [review]
AST looks good to me 👍

File: lib/live_view_native/swiftui/rules_parser/post_processors.ex (hunk 1 above)
Comment (NduatiK) [reply lib/live_view_native/swiftui/rules_parser/post_processors.ex]: Removed, thanks

**Review by carson-katri — APPROVED** [review]

**Pull request merged**